### PR TITLE
Create Using Obsidian as a TTRPG Campaign Manager

### PR DESCRIPTION
**Pull request by ObsidianTTRPGProject**
Creating new note on my YouTube Channel that teaches people how to use Obsidian as a TTRPG Campaign Manager. 

- Added/Edited note ...

## Checklist

- [X] I have renamed all attached images with descriptive file names (or I did not include any images)
- [X] Before creating a new note, I searched the vault (or I only edited existing notes)
- [X] (Optional) In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).

### CODE REVIEW

**Comment by ObsidianTTRPGProject** [comment]
Don't seem to have access to add pictures. These are the files referenced in this note. 

TTRPG Campaign Mgt - FacebookTTRPGUserGroup
TTRPG Campaign Mgt - Where To Begin

File: 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+aliases: 
+- Josh Plunkett
+- Obsidian TTRPG
```
Comment (claremacrae) [inline 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager:4]: I'm not confident of these aliases, but should a specific page be given a much more general alias - will that cause confusion, perhaps if we have other pages about the more general Obsidian TTRPG topic?

File: 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+tags:
+- seedling
+publish: true
+---
+# Josh's YouTube Obsidian TTRPG Playlist
+Link: [YouTube](https://youtube.com/playlist?list=PLV5XWfKkFpk7MJTKv5YdSSpT9b-vLslWu) / [Patreon](https://www.patreon.com/JPlunkett)
+
+Hosted by [[Josh Plunkett|Joshua Plunkett]].
+
+Looking for the perfect TTRPG Campaign Manager? Looks no further!
```
Comment (claremacrae) [inline 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager:16]: Suggest "Look no further"

File: 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (claremacrae) [inline 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager:1]: Please rename this note to add `.md` to the filename

**Review by claremacrae — CHANGES_REQUESTED** [review]
Please rename this file to add the `.md` file extension. Thank you.

File: 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager (hunk 3 above)
Comment (claremacrae) [reply 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager:1]: oops - sorry for the duplicate comment

File: 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager (hunk 1 above)
Comment (ObsidianTTRPGProject) [reply 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager]: I've removed them. Just had my social media hat on where I tag everything to trigger the Google search algorithms.

File: 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager (hunk 2 above)
Comment (ObsidianTTRPGProject) [reply 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager]: Done

File: 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager (hunk 3 above)
Comment (ObsidianTTRPGProject) [reply 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager]: Done :)

File: 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager (hunk 1 above)
Comment (claremacrae) [reply 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager]: Thanks!

File: 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+tags:
+- seedling
+publish: true
+---
+# Josh's YouTube Obsidian TTRPG Playlist
+Link: [YouTube](https://youtube.com/playlist?list=PLV5XWfKkFpk7MJTKv5YdSSpT9b-vLslWu) / [Patreon](https://www.patreon.com/JPlunkett)
+
+Hosted by [[Josh Plunkett|Joshua Plunkett]].
+
+Looking for the perfect TTRPG Campaign Manager? Looks no further.
```
Comment (claremacrae) [inline 04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager.md:13]:
```suggestion
Looking for the perfect TTRPG Campaign Manager? Look no further.
```

**Review by claremacrae — APPROVED** [review]
Looks good to me!

**Pull request merged**